src/views/Pools/components/LockedPool/Modals/ExtendDurationModal.tsx

```diff
@@ -39,10 +47,16 @@ const ExtendDurationModal: React.FC<ExtendDurationModal> = ({
         isOverMax: totalDuration > MAX_LOCK_DURATION,
       }
     },
-    [currentLockedAmount, currentDuration],
+    [currentLockedAmount, lockEndTime],
   )
 
-  const prepConfirmArg = useCallback(({ duration }) => ({ finalDuration: duration, finalLockedAmount: 0 }), [])
+  const prepConfirmArg = useCallback(
+    ({ duration }) => ({
+      finalDuration: duration,
+      finalLockedAmount: currentBalance ? getBalanceAmount(MIN_AMOUNT, stakingToken.decimals).toNumber() : 0,
```

Comment:
should it be `currentBalance.gt(MIN_AMOUNT) ? ... : ...`  ?

Reply:
Done